.github/workflows/helm-release-chart.yml

```diff
@@ -0,0 +1,43 @@
+name: Release Helm Chart
+on:
+  push:
+    tags:
+     - v*
+    paths:
+    - chart/**
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Helm
+      uses: azure/setup-helm@v4
+
+    - name: Log in to GHCR
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Update Helm Dependencies
+      working-directory: chart
+      run: |
+        helm dependency update
+
+    - name: Package Helm Chart
+      working-directory: chart
+      run: |
+        helm package .
+        mv *.tgz ../packaged-chart.tgz
+
+    - name: Push Helm Chart to GHCR
+      env:
+        HELM_EXPERIMENTAL_OCI: 1
```

Comment:
I don't think this env variable is needed anymore with recent versions of helm

https://helm.sh/docs/topics/registries/#enabling-oci-support-prior-to-v380